torchvision/datasets/video_utils.py

```diff
@@ -371,10 +376,10 @@ def __getstate__(self):
         video_pts = [x.to(torch.int64) for x in self.video_pts]
         # video_pts can be an empty list if no frames have been decoded
         if video_pts:
-            video_pts = torch.cat(video_pts)
+            video_pts = torch.cat(video_pts)  # type: ignore[assignment]
```

Comment:
Were you getting warnings here? Why this type ignore?

Reply:
Before this branch `video_opts` is a `List[Tensor]`, but now becomes a `Tensor`.